teuthology/orchestra/run.py

```diff
@@ -102,6 +102,15 @@ def wait(self):
 
         status = self._get_exitstatus()
         self.exitstatus = self.returncode = status
+        for stream in ('stdout', 'stderr'):
+            if hasattr(self, stream):
+                stream_obj = getattr(self, stream)
+                # Despite ChannelFile having a seek() method, it raises
+                # "IOError: File does not support seeking."
```

Comment:
I guess it's friendlier/points more to the error than just a traceback with attr not found.  go paramiko.